collectors/python.d.plugin/exim/README.md

```diff
@@ -3,6 +3,27 @@
 Simple module executing `exim -bpc` to grab exim queue.
 This command can take a lot of time to finish its execution thus it is not recommended to run it every second.
 
+**Requirements**
```

Comment:
make it like there
https://github.com/netdata/netdata/tree/master/collectors/python.d.plugin/adaptec_raid#requirements

Reply:
What is wrong with my file? :)
I checked above link, looks similar.

Reply:
`**Requirements**` => `## Requirements`

CLA problems btw :trollface:

Reply:
I don't know how to solve CLA problem. I removed PR and create new one, but looks like problem was because issue open from different account. I don't know how to solve that.
I will solve this heading problem now.

Reply:
> because issue open

because PR open is open from @petarkozic commits from @kozicpetar

Reply:
but I removed PR and create new one from @petarkozic and commit from @petarkozic

Reply:
Your commits are still associated with your @kozicpetar account. If you click on the **Commit** tab you can see each commit has a different user icon than the one you have when you're chatting with us here. I think it has to do with the email address on this GitHub account and the email address you have configured with `git config --global user.email "email@example.com"` not matching properly.

https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address

Reply:
All that is set on right way, I checked already. Problem is because first commit I do from wrong account, but that PR I was removed.

Ok, how can I solve this?
What I need to remove that I can commit from only one account.